### ISSUE TITLE
v0.7 breaks artifactory for Python <3.6

### ISSUE THREAD

**Issue by hagenw** [devopshq/artifactory#130]
The introduced import of [secrets](https://docs.python.org/3/library/secrets.html) ([after v0.6](https://github.com/devopshq/artifactory/compare/v0.6...develop)) breaks the package for versions of Python <3.6.

E.g. on a CI job with `dohq-artifactory-0.7.297` we get
```
/usr/local/lib/python3.5/site-packages/dohq_artifactory/admin.py:2: in <module>
    import secrets
E   ImportError: No module named 'secrets'
```

**Comment by nikolasj** [comment]
@hagenw Please check with the new version `dohq-artifactory-0.7.308`

**Comment by hagenw** [comment]
Thanks for the very fast fix, it is now working again with Python 3.5.
